docs/operating-scylla/procedures/cluster-management/create-cluster-multidc.rst

```diff
@@ -67,7 +67,7 @@ The file can be found under ``/etc/scylla/``.
 - **seeds** - Specify the IP of the node you chose to be a seed node. New nodes will use the IP of this seed node to connect to the cluster and learn the cluster topology and state.
 - **listen_address** - IP address that the Scylla use to connect to other Scylla nodes in the cluster
 - **endpoint_snitch** - Set the selected snitch
-- **rpc_address** - Address for client connection (Thrift, CQLSH)
+- **rpc_address** - Address for client connection, for instance, CQLSH
```

Comment:
This should probably say CQL, not "for instance cqlsh".

Reply:
sure. more consistent this way as well.